specificatie/openapi.yaml

```diff
@@ -0,0 +1,377 @@
+    NationaliteithistorieHalCollectieEmbedded:
+      type: object
+      properties:
+        nationaliteithistorie:
+          type: array
+          items:
+            $ref: '#/components/schemas/Nationaliteithistorie'
+    Nationaliteithistorie:
+      allOf:
+      - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.0.0/specificatie/openapi.yaml#/components/schemas/Nationaliteit"
+      - description: |
+                      Gegevens over de verblijfsrechtelijke status van de ingeschrevene.
+                      * **datumEinde**: Datum waarop de geldigheid van de gegevens over de Nationaliteit is beëindigd.
+                      * **datumIngang**: Datum waarop de gegevens over de Nationaliteit geldig zijn geworden.
+                      * **aanduiding** : Nationaliteittabel, die aangeeft over welke verblijfsrechtelijke status de ingeschrevene beschikt.
+        properties:
+          geheimhoudingPersoonsgegevens:
+            type: boolean
+            title: "Indicatie geheim"
+            description: |
+                          Een aanduiding die aangeeft dat gegevens wel of niet verstrekt mogen worden.
+                          Indien true : op verzoek van deze persoon is het verstrekken van gegevens over deze persoon aan bepaalde derden niet toegestaan.
+          datumEindeGeldigheid:
+            $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/DatumOnvolledig"
+          redenBeeindigen:
+            $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+          indicatieNationaliteitBeeindigd:
+            type: boolean
+            description: |
+                          Een aanduiding dat de nationaliteit beëindigd is.
+            example: true
+    VerblijfplaatshistorieHalCollectie:
+      type: object
+      properties:
+        _links:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks"
+        _embedded:
+          $ref: '#/components/schemas/VerblijfplaatshistorieHalCollectieEmbedded'
+    VerblijfplaatshistorieHalCollectieEmbedded:
+      type: object
+      properties:
+        verblijfplaatshistorie:
+          type: array
+          items:
+            $ref: '#/components/schemas/VerblijfplaatshistorieHal'
+    VerblijfplaatshistorieHal:
+      allOf:
+        - $ref: '#/components/schemas/Verblijfplaatshistorie'
+        - properties:
+            _links:
+              $ref: "#/components/schemas/VerblijfplaatshistorieLinks"
+    Verblijfplaatshistorie:
+      allOf:
+      - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.0.0/specificatie/openapi.yaml#/components/schemas/Verblijfplaats"
+      - properties:
```

Comment:
is hier expres de description weggelaten?

Reply:
Nee, blijkbaar was die er niet. Ik ben me er tenminste niet van bewust dat ik deze heb verwijderd. Nu toegevoegd.